scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,48 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    """
+    Khatri-rao product
+
+    Parameters
+    ----------
+    a:  (I, K) ndarray
+        Input array
+    b:  (J, K) ndarray
+        Input array
+
+    Returns
+    -------
+    C:  ((IJ), K) ndarray
+        Khatri-rao product of 'a' and 'b'.
+
+    Notes
+    -----
+    Mathematically:
+
+    A * B = (A_{ij}  \bigotimes B_{ij})_{ij}
```

Comment:
This will not be rendered correctly, should be:
```
.. math::

    A * B = (A_{ij}  \bigotimes B_{ij})_{ij}
```
Also, `A * B` doesn't belong here right? - the definition is the right-hand side?

Reply:
According to Wikipedia:

![Screenshot_20200202_201900](https://user-images.githubusercontent.com/57490926/73611652-2b121500-45f9-11ea-9652-c8f705e9a4dc.png)

Reply:
Yeah, I'd say that Wikipedia is confusing. It's a product, not an equation. Just keeping the right-hand side here looks like the way to go.

Reply:
Using `:=` can help most users as a definition symbol

Reply:
I would also add more readable programmatic definition to Ralf's suggestion

```
.. math::

    A * B := (A_{ij}  \bigotimes B_{ij})_{ij}

which is the Kronecker product of every column of A and B, e.g., 

    c = np.vstack([np.kron(a[:, k], b[:, k]) for k in range(b.shape[1])]).T
```

Reply:
This suggestion is the one comment not addressed yet, seems good to add.

Reply:
I added it